### PR TITLE
Added snippets for synchronous APIs and finished last of the snippets.

### PR DESCRIPTION
**Pull request by LarryOsterman**
# Added Snippets for the remaining attestation service APIs.

Cleans up the attestation service `WithResponse` APIs to better conform to API guidelines.
Adds snippets for all of the remaining attestation APIs.

# All SDK Contribution checklist:
- [X] **The pull request does not introduce [breaking changes]**
- [X] **CHANGELOG is updated for new features, bug fixes or other significant changes.**
- [X] **I have read the [contribution guidelines](https://github.com/Azure/azure-sdk-for-java/blob/main/CONTRIBUTING.md).**

## [General Guidelines and Best Practices](https://github.com/Azure/azure-sdk-for-net/blob/main/CONTRIBUTING.md#general-guidelines)
- [X] Title of the pull request is clear and informative.
- [X] There are a small number of commits, each of which have an informative message. This means that previously merged commits do not appear in the history of the PR. For more information on cleaning up the commits in your PR, [see this page](https://github.com/Azure/azure-powershell/blob/master/documentation/development-docs/cleaning-up-commits.md).

### [Testing Guidelines](../../CONTRIBUTING.md#building-and-unit-testing)
- [X] Pull request includes test coverage for the included changes.

### CODE REVIEW

**Comment by LarryOsterman** [comment]
Ping :).

File: sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAdministrationAsyncClient.java [hunk 1]
```diff
@@ -759,8 +826,35 @@ private AttestationToken generatePolicySetToken(String policy, AttestationSignin
...
+     * <p><strong>Add a new certificate to the set of policy management certificates for this instance.</strong></p>
+     * <!-- src_embed com.azure.security.attestation.AttestationAdministrationAsyncClient.removePolicyManagementCertificate -->
+     * <pre>
+     * Mono&lt;PolicyCertificatesModificationResult&gt; removeResultMono = client.removePolicyManagementCertificate&#40;
+     *     new PolicyManagementCertificateOptions&#40;certificateToAdd, new AttestationSigningKey&#40;certificate, privateKey&#41;&#41;&#41;;
+     * removeResultMono.subscribe&#40;removeResult -&gt; System.out.printf&#40;&quot;Result: %s&#92;n&quot;, removeResult.getCertificateResolution&#40;&#41;.toString&#40;&#41;&#41;&#41;;
+     * </pre>
+     * <!-- end com.azure.security.attestation.AttestationAdministrationAsyncClient.removePolicyManagementCertificate -->
+     *
+     * <p><strong><i>Note:</i></strong> It is not considered an error to removethe same certificate twice. If
```
Comment (pallavit) [inline sdk/attestation/azure-security-attestation/src/main/java/com/azure/security/attestation/AttestationAdministrationAsyncClient.java:854]: [nit] space

File: sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation/AttestationClientJavaDocCodeSnippets.java [hunk 2]
```diff
@@ -245,8 +248,8 @@ public static void attestOpenEnclaveAsync1() {
         // END: com.azure.security.attestation.AttestationAsyncClient.getOpenIdMetadata
 
         // BEGIN: com.azure.security.attestation.AttestationAsyncClient.getAttestationSigners
-        Mono<List<AttestationSigner>> signers = client.listAttestationSigners();
-        signers.block().forEach(cert -> {
```
Comment (pallavit) [inline sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation/AttestationClientJavaDocCodeSnippets.java:249]: >block [](http://example.com/codeflow?start=16&length=5)

Thank you for updating this. Not sure, how I missed it in the last PR.

**Review by pallavit — APPROVED** [review]
:shipit:

File: sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation/AttestationClientJavaDocCodeSnippets.java (hunk 2 above)
Comment (LarryOsterman) [reply sdk/attestation/azure-security-attestation/src/samples/java/com/azure/security/attestation/AttestationClientJavaDocCodeSnippets.java:249]: Yeah - I was running through the samples and noticed that the async samples had .block in them - SLOPPY!

**Pull request merged**